FaunaDB.Client/Client/FaunaClient.cs

```diff
@@ -122,6 +122,13 @@ public FaunaClient(IClientIO clientIO)
 
             RaiseForStatusCode(responseHttp);
 
+            if (responseHttp.ResponseHeaders.ContainsKey("X-Txn-Time")) {
+                // there shouldn't ever be more than one...
```

Comment:
what?

Reply:
More than one X-Txn-Time. Too vague?

Reply:
doesn't `responseHttp.ResponseHeaders` represent all the response headers?

Reply:
Hmm. I interpreted the type to be a dictionary with string -> enum[string]...

Reply:
.NET apparently supports multiple-value headers? Weird. https://msdn.microsoft.com/en-us/library/system.net.http.headers.httpheaders(v=vs.118).aspx